src/trace.rs

```diff
@@ -44,30 +44,32 @@ pub fn init(color: bool, json: bool, levels: &str) {
     // This is a hidden and undocumented functionality.
     let metrics_layer_enabled = metrics_layer_enabled();
 
+    let subscriber = tracing_subscriber::registry::Registry::default()
+        .with(tracing_subscriber::filter::EnvFilter::from(levels));
+
     let dispatch = if json {
```

Comment:
I wanted to try to refactor this because there is a quite a bit of duplication, but I struggled to satisfy the complier due to all of the generics used in tracing-subscriber. This duplication is already there in the current implementation though.

Reply:
I'd be very interested in eavesdropping on that if the timezones can work..

Reply:
👍 I would definitely learn from pairing on it.

Reply:
Just dropping a note for posterity that we attempted to refactor this, but it quickly grew complex enough that it didn't seem worth it for the small amount of duplication (we started adding in a new wrapping, generic, type around the fmt layers).